pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java

```diff
@@ -566,4 +572,9 @@ ClientBuilder authentication(String authPluginClassName, Map<String, String> aut
      * @return
      */
     ClientBuilder socks5ProxyPassword(String socks5ProxyPassword);
+
+    /**
+     * Disable The "auto release useless connections" feature.
+     */
+    ClientBuilder disabledAutoReleaseUselessConnections();
```

Comment:
Can we only use `connectionMaxIdleSeconds`? -1 means the auto release is disabled.

Reply:
> Can we only use connectionMaxIdleSeconds? -1 means the auto release is disabled.

Good idea. Already fixed.